DHL Tracking - Deprecate (Independent Publisher)

last72 opened the pull request:
As requested by DHL

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

last72 committed:
Deprecate and contact changed

vmanoharas commented:
Hi @last72, 

If you wish to deprecate this connector, kindly add deprecate keyword to all the actions and triggers.  Please refer the documentation here - https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation

last72 committed:
Operation deprecated

Operation deprecated as per https://learn.microsoft.com/en-us/connectors/custom-connectors/deprecate-operation

last72 commented:
@vmanoharas Operation deprecated as per documentation

vmanoharas commented:
Thank you @last72, could you please address the second swagger validation error? Please ignore the first one. 

##[error]The 'title' value must end with alphanumeric character. ValuePath: /info/title
##[error]The connector metadata property 'Privacy policy' is required. ValuePath: /x-ms-connector-metadata

last72 commented:
Hi @vmanoharas, Privacy Policy has been removed as per requested by DHL. Can I place an empty string "" or a placeholder text?

vmanoharas commented:
> Hi @vmanoharas, Privacy Policy has been removed as per requested by DHL. Can I place an empty string "" or a placeholder text?

Yes kindly add placeholder test @last72

last72 committed:
Privacy policy placeholder

last72 commented:
@vmanoharas I have updated to include placeholder. I was unable to validate swagger using paconn

vmanoharas reviewed (approved):
Dear Partner

I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks.
Please let us know if you have any questions.
Thank you very much for working with us.

vmanoharas commented:
[[certify-connector]]

vmanoharas commented:
Hello Partner,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, 
Thank you very much for working with us.
